packages/dashboard/src/pages/Personas/components/ContactTableRow/index.tsx

```diff
@@ -0,0 +1,143 @@
+        )
+    }, [])
+
+    return (
+        <ContactTableRowUI
+            handleClickInvite={handleClickInvite}
+            handleClickStar={handleClickStar}
+            theme={theme}
+            contact={contact}
+            index={index}
+        />
+    )
+})
+
+export interface ContactTableRowUIProps extends Omit<ContactTableRowProps, 'network'> {
+    handleClickInvite(): void
+    handleClickStar(): void
+    theme: 'light' | 'dark'
+}
+
+export const ContactTableRowUI = memo<ContactTableRowUIProps>(
+    ({ contact, index, handleClickStar, handleClickInvite, theme }) => {
+        const t = useDashboardI18N()
+        const classes = useStyles()
+        const [first, last] = contact.name.split(' ')
+
+        return (
+            <TableRow>
+                <TableCell align="left" variant="body">
+                    <Box display="flex" alignItems="center">
+                        <Typography>{index}</Typography>
+                        <Box className={classes.favorite}>
+                            {contact.fingerprint ? (
+                                <StarIcon
+                                    sx={{
+                                        fill: contact.favorite ? MaskColorVar.warning : MaskColorVar.iconLight,
+                                        cursor: 'pointer',
+                                    }}
+                                    onClick={handleClickStar}
+                                />
+                            ) : null}
+                        </Box>
+                        <Box className={classes.avatarContainer}>
+                            <Avatar
+                                aria-label={contact.name}
+                                src={contact.avatar}
+                                sx={{
+                                    backgroundColor: mapContactAvatarColor(contact.identifier.toText(), theme),
+                                    width: 48,
+                                    height: 48,
+                                }}>
+                                {first[0]}
+                                {(last || '')[0]}
+                            </Avatar>
+                            {contact.fingerprint ? <MaskNetworkIcon className={classes.maskIcon} /> : null}
+                        </Box>
+                        <Box className={classes.info}>
+                            <Typography>{contact.name}</Typography>
+                            <Typography className={classes.userId}>@{contact.identifier.userId}</Typography>
+                        </Box>
+                    </Box>
+                </TableCell>
+                <TableCell align="center">
+                    {!contact.fingerprint ? (
+                        <Button color="secondary" size="small" className={classes.button} onClick={handleClickInvite}>
+                            {t.personas_contacts_invite()}
+                        </Button>
+                    ) : null}
```

Comment:
Why not add a text placeholder in the cell?

Reply:
Not required in design